app/src/main/java/org/sopt/dosopttemplate/HomeFragment.kt

```diff
@@ -0,0 +1,50 @@
+package org.sopt.dosopttemplate
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import org.sopt.dosopttemplate.databinding.FragmentHomeBinding
+import androidx.recyclerview.widget.ConcatAdapter
+import androidx.recyclerview.widget.LinearLayoutManager
+
+class HomeFragment : Fragment() {
+    private var _binding: FragmentHomeBinding? = null
+    private val binding: FragmentHomeBinding
+        get() = requireNotNull(_binding) { "바인딩 객체가 생성되지 않았습니다." }
+    private val viewModel by viewModels<HomeViewModel>()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? { // 이제 반환하는 View가 Null일 수 없으므로 ? 지워도 됨
+        _binding = FragmentHomeBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val myAdapter = MyAdapter(requireContext())
+        binding.rvFriends.adapter = myAdapter
+        myAdapter.setFriendList(viewModel.mockMyList)
+
+        val friendAdapter = FriendAdapter(requireContext())
+        binding.rvFriends.adapter = friendAdapter
+        friendAdapter.setFriendList(viewModel.mockFriendList)
+
+        // concatAdapter 이용해 내 프로필과 친구 목록 연결
+        val concatAdapter = ConcatAdapter(myAdapter, friendAdapter)
+        binding.rvFriends.adapter = concatAdapter
```

Comment:
```suggestion
        val myAdapter = MyAdapter(requireContext())
        myAdapter.setFriendList(viewModel.mockMyList)

        val friendAdapter = FriendAdapter(requireContext())
        friendAdapter.setFriendList(viewModel.mockFriendList)

        // concatAdapter 이용해 내 프로필과 친구 목록 연결
        val concatAdapter = ConcatAdapter(myAdapter, friendAdapter)
        binding.rvFriends.adapter = concatAdapter
```

rvFriends의 adapter로 concatAdapter를 사용하고 싶은 거니까 마지막에만 연결해주면 됩니다 !